index.js

```diff
@@ -1,109 +1,145 @@
 #!/usr/bin/env node
 
-import fs from 'fs';
-import path from 'path';
+import 'babel-polyfill';
+import program from 'commander';
+import { stat, readFile, writeFile, unlink } from 'fs-extra';
 import { compileToString } from 'node-elm-compiler';
+import path from 'path';
 import sh from 'shelljs';
 
-const filename = (process.argv[2] || '').trim();
+(async () => {
+  program
```

Comment:
`commander` npm module (from where we import `program`) gives us an easy way of defining command line args. My PR only supports one argument, but if you want some data to go down to elm like in v2, that should be easy to arrange.